doc/minversions.md

```diff
@@ -1,3 +1,20 @@
+---
+myst:
+  # Customized URI schemes that are converted to external links
```

Comment:
Hm. Based on how I understand the documentation so far and being consistent with `url_schemes:`, using "URL" seem to be better, here. If we use "URI" I feel it can make sense to introduce this shortcut.

```suggestion
  # Customized URL schemes that are converted to external links
```

Reply:
For me it seems URI is still the right term here since an URL is already a link (a location) while an URI is just the identifier that is then converted to an URL.

Reply:
The MyST documentation uses URI: https://myst-parser.readthedocs.io/en/latest/configuration.html

For the difference between URI and URL, see https://www.godaddy.com/resources/ae/skills/uri-vs-url.

I agree with @michaelgrund that URI is the right term.

Reply:
Hm, looks like the MyST documentation is not 100 % consistent here 🙃:

at https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#customising-external-url-resolution
> To specify a custom list of URL schemes, you can set the myst_url_schemes configuration option. By default this is set to ["http", "https", "ftp", "mailto"].

but at https://myst-parser.readthedocs.io/en/latest/configuration.html
> myst_url_schemes  |  URI schemes that are converted to external links (default: {'http': None, 'https': None, 'mailto': None, 'ftp': None})

Reply:
Anyway, it's not one comment line and it doesn't hurt if it's correct or not.

Reply:
Reading https://www.godaddy.com/resources/ae/skills/uri-vs-url
> URIs can be further divided into two categories: URLs and URNs. [...]
> The key difference is: all URLs are URIs, but not all URIs are URLs. 

So, let's keep URIs here.